integration-tests/core-support/deployment/pom.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@lburgazzoli this looks like an extension. Are you sure you wanted to add this under integration-tests?